mlrun/runtimes/base.py

```diff
@@ -714,8 +714,22 @@ def _submit_job(self, runspec, schedule, db, watch):
             txt = get_in(resp, "status.status_text")
             if txt:
                 logger.info(txt)
+        # watch is None only in scenario where we run a pipeline step, in this case we don't want to watch the run logs
+        # but rather just pull the state of the run from the DB, and print the logs once the task is done
+        if (
+            watch is None
+            and self.kfp
+            and config.httpdb.pipelines.logs.pull_state.mode == "enabled"
+        ):
+            state_interval = config.httpdb.pipelines.logs.pull_state.pull_state_interval
+            logs_interval = config.httpdb.pipelines.logs.pull_state.pull_logs_interval
+
+            runspec.wait_for_completion(
+                show_logs=True, sleep=state_interval, logs_interval=logs_interval
```

Comment:
According to the above comment where we don't want the logs, shouldn't this be `show_logs=False`?